### PR TITLE
Add Kate to IoT Edge WG leadership

### PR DESCRIPTION
**Pull request by kate-goldenring**
Changes IoT Edge WG leadership swapping Kate in for @cindyxing as a co-chair

FYI: @dejanb and @cantbewong

### CODE REVIEW

**Review by CindyXing — APPROVED** [review]
This sounds great! Thanks

Look forward to this group making great impact in the industry.

**Comment by dejanb** [comment]
+1 from my side

File: sigs.yaml [hunk 1]
```diff
@@ -3001,12 +3001,12 @@ workinggroups:
...
-    - github: cindyxing
-      name: Cindy Xing
-      company: Microsoft
     - github: dejanb
       name: Dejan Bosanac
       company: Red Hat
+    - github: kate-goldenring
+      name: Kate Goldenring
+      company: Microsoft
     emeritus_leads:
```
Comment (dejanb) [inline sigs.yaml:3010]: We should add Cindy to emeritus section

Comment (cantbewong) [reply sigs.yaml:3010]: yes I agree that Cindy deserves to go on the emeritus list unless she has some objection

**Comment by mrbobbytables** [comment]
/close
The IoT edge group is moving over to the CNCF

**Pull request closed without merging**